libs/ui/packages/native/src/components/cta/Button/index.tsx

```diff
@@ -33,12 +34,21 @@ const IconContainer = styled.View<{
     p.iconButton ? "" : p.iconPosition === "left" ? `margin-right: 10px;` : `margin-left: 10px;`}
 `;
 
-export const Base = baseStyled(TouchableOpacity).attrs<ButtonProps>((p) => ({
-  ...getButtonColorStyle(p.theme.colors, p).button,
-  // Avoid conflict with styled-system's size property by nulling size and renaming it
-  size: undefined,
-  sizeVariant: p.size,
-}))<
+export const Base = baseStyled(TouchableOpacity).attrs<ButtonProps>((p) => {
```

Comment:
If design system is happy with this change, I think we need to apply the same change to libs/ui/package/react.

Reply:
Sent them a message on slack, so just waiting to see what they say 👍

Reply:
Super. I think as Olivier has approve it they are mostly happy, please do not forget align LLD component any basic design will be good on my side but you can ask Diego.